### ISSUE TITLE
Interacting with blocks in dimensions other than the Overworld causes disconnections

### ISSUE THREAD

**Issue by ikafly144** [cerus/maps#13]
### Issue description

```
[13:24:56] [Server thread/INFO]: ikafly144 lost connection: Internal Exception: java.lang.IllegalArgumentException: Cannot measure distance between world_nether and world
[13:24:56] [Server thread/INFO]: ikafly144 left the game
```
https://github.com/cerus/maps/blob/9cef0e99be2383b4c979f50578212a86b04acaea/plugin/src/main/java/dev/cerus/maps/plugin/listener/PlayerListener.java#L58
This line seems to be the cause

### To reproduce

Place a map screen in the overworld.
Go to a dimension other than the Overworld (such as the Nether) and right-click on a block.

### Expected behaviour

Right clicking doesn't kick you off the server

**Comment by cerus** [comment]
Thank you for reporting this bug, I'll try to get a fix out asap. Until the bug is fixed you could try setting `use-triangulation` in `plugins/maps/maps-config.yml` to `false` - this should fix the issue for now.

**Comment by cerus** [comment]
Fixed in 84ad5a16e70fdd3b399296c7ca5fc2dd53823a6b (see release 3.8.7)
